control_plane/src/broker.rs

```diff
@@ -0,0 +1,36 @@
+use anyhow::Context;
+
+use std::path::PathBuf;
+
+use crate::{background_process, local_env};
+
+pub fn start_broker_process(env: &local_env::LocalEnv) -> anyhow::Result<()> {
+    let broker = &env.broker;
+    let listen_addr = &broker.listen_addr;
+
+    print!("Starting neon broker at {}", listen_addr);
+
+    let args = [format!("--listen-addr={listen_addr}")];
+
+    background_process::start_process(
+        "storage_broker",
+        &env.base_data_dir,
+        &env.storage_broker_bin(),
+        &args,
+        [],
+        background_process::InitialPidFile::Create(&storage_broker_pid_file_path(env)),
+        || {
+            Ok(true) // TODO
```

Comment:
We have a `GET /metrics` endpoint now and this module has `requwest` as a dependency, let's wait for that properly?
Should be simple, since that's how we essentially tested `etcd` before:

```rust
let client = reqwest::blocking::Client::new();

//////

|| {
    for broker_endpoint in &etcd_broker.broker_endpoints {
        let request = broker_endpoint
            .join("health")
            .with_context(|| {
                format!(
                    "Failed to append /health path to broker endopint {}",
                    broker_endpoint
                )
            })
            .and_then(|url| {
                client.get(&url.to_string()).build().with_context(|| {
                    format!("Failed to construct request to etcd endpoint {url}")
                })
            })?;
        if client.execute(request).is_ok() {
            return Ok(true);
        }
    }

    Ok(false)
},
```

Reply:
> Should be simple, since that's how we essentially tested etcd before:

That code was actually broken, I rewrote it.